README.md

```diff
@@ -295,19 +295,21 @@ HTTP/1.1 204 No Content
 Request:
 
 ```sh
-curl --include --request POST http://localhost:3000/sign-up \
+curl http://localhost:4741/sign-up \
+  --include \
+  --request POST \
   --header "Content-Type: application/json" \
   --data '{
     "credentials": {
-      "email": "an@example.email",
-      "password": "an example password",
-      "password_confirmation": "an example password"
+      "email": "'"${EMAIL}"'",
```

Comment:
Thanks, @gaand. I think we should revisit this in the future. A suggestion doesn't come immediately to mind.

My first thought was to document the scripts maintained in the `scripts` directory, but it doesn't accomplish the goal of showing it in the README. Perhaps a `How to Use This Documentation` section? Not sure I like that.